tests/flow/test_ts_madd.py

```diff
@@ -98,3 +98,19 @@ def test_extensive_ts_madd():
         for pos,datapoint in enumerate(returned_floats,start=1):
             assert pos == datapoint[0]
             assert float_lines[pos-1] == float(datapoint[1])
+
```

Comment:
Can you explain how the test verifies that the wrong command is not replicated to the replica?